docker/Dockerfile.base

```diff
@@ -40,7 +40,7 @@ RUN mkdir -m0755 /licenses
 COPY ./LICENSE-Apache /licenses/apache.txt
 
 ### Add necessary Red Hat repos and packages
-RUN INSTALL_PKGS="iproute procps" && \
+RUN INSTALL_PKGS="iproute procps shadow-utils" && \
```

Comment:
I think our security posture is reduced if we leave the shadow-utils package installed for use at runtime. Could we uninstall it after creating the group?